packages/commonwealth/client/scripts/controllers/server/contracts.ts

```diff
@@ -226,6 +226,61 @@ class ContractsController {
     return this._store.add(contract);
   }
 
+  // TODO there is similar function above => "add"
```

Comment:
Lets not remove it yet- @0xdantrinh may have had plans for this. But definitely use this addContractAndAbi function for all our bet1 work.

Reply:
You can remove it as long as it isn't being referenced anywhere. Which I think it isn't!